integration/src/main/java/com/aws/greengrass/mqtt/moquette/ClientDeviceAuthorizer.java

```diff
@@ -62,20 +63,43 @@ public boolean checkValid(String clientId, String username, byte[] password) {
                     + "client ID and has a valid AWS IoT client certificate.");
             return false;
         }
-
-        boolean canConnect = canDevicePerform(sessionId, "mqtt:connect", "mqtt:clientId:" + clientId);
+        boolean canConnect = false;
+        try {
+            canConnect = canDevicePerform(sessionId, "mqtt:connect", "mqtt:clientId:" + clientId);
+        } catch (InvalidSessionException e) {
+            LOG.error("{}: {}", e.getClass().getSimpleName(), e.getMessage());
```

Comment:
do we need this log?  retrying is part of expected behavior now, so error would alert users to something that may not be an error. we already have the error log later on if the retry of canDevicePerform fails, which would be enough I think

Reply:
Yeah that's probably fair. I'd just log at debug that it happened, perhaps that would be meaningful for some reason